xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/panels.less

```diff
@@ -55,4 +55,10 @@ a.panel-collapse-carret {
 .panel-radio {
   float: left;
   padding: 15px
-}
\ No newline at end of file
+}
+
+// Checkbox xhidden in the Edit Panel (Page Information) ==================
+#editPanels #xhidden {
+  position: relative;
+  margin-left: -20px;
```

Comment:
No, this is not the right fix. Now that you added the ``xform`` class on the edit form with #1247 you only need to remove the

    <span class="checkbox">

Reply:
Oh alright, working on it.